xml/installation-installation-ironic-ironic_provisioning.xml

```diff
@@ -76,9 +76,9 @@
    <para>
     The use of <literal>agent_</literal> drivers is predicated upon Glance
     images being backed by a Swift image store, specifically the need for the
-    temporary file access features. Use of Ceph or the filesystem as a Glance
-    back-end image store means that the <literal>agent_</literal> drivers
-    cannot be used.
+    temporary file access features. Using the filesystem as a Glance back-end
+    image store means that the <literal>agent_</literal> drivers cannot be
+    used.
```

Comment:
If you reformat things that you are not actually changing, that unfortunately makes it harder to review your commits. Please try to avoid that in the future. (If formatting is too terrible, then first make a commit that changes the formatting only and then do your content-change commit.)